actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -36,21 +36,23 @@ module Redirecting
     # If you are using XHR requests other than GET or POST and redirecting after the
     # request then some browsers will follow the redirect using the original request
     # method. This may lead to undesirable behavior such as a double DELETE. To work
-    # around this  you can return a <tt>303 See Other</tt> status code which will be
+    # around this you can return a <tt>303 See Other</tt> status code which will be
     # followed using a GET request.
     #
     #   redirect_to posts_url, status: :see_other
     #   redirect_to action: 'index', status: 303
     #
-    # It is also possible to assign a flash message as part of the redirection. There are two special accessors for the commonly used flash names
+    # It is also possible to assign a flash message as part of the redirection.
+    # There are two special accessors for the commonly used flash names
```

Comment:
@shime can you revert these two line wrapping changes? Then we are good to merge.